Dockerfile

```diff
@@ -1,20 +1,25 @@
-FROM alpine:3.8 AS build
+FROM ubuntu:14.04 AS build
```

Comment:
Why 14.04?  According to https://wiki.ubuntu.com/Releases, it just went out of support.

Reply:
ohhh I didn't check Ubuntu's wiki. I just went for the version I considered most stable from the Docker's supported Ubuntu image list [https://hub.docker.com/_/ubuntu/](https://hub.docker.com/_/ubuntu/). Version 19.04 was just released this month. Do you think we should go for a more stable one? Version 18.04 maybe? From [https://hub.docker.com/_/ubuntu/]():
## Supported tags and respective Dockerfile links
*    18.04, bionic-20190424, bionic, latest (bionic/Dockerfile)
*   18.10, cosmic-20190418, cosmic (cosmic/Dockerfile)
*    19.04, disco-20190423, disco, rolling (disco/Dockerfile)
*    14.04, trusty-20190425, trusty (trusty/Dockerfile)
*    16.04, xenial-20190425, xenial (xenial/Dockerfile)

Reply:
I would go with 18.04, the even numbered releases are LTS (Long Term Support). As per https://wiki.ubuntu.com/Releases 19.04 goes off support in 2020, while 18.04 is supported until 2023. I am not sure why 18.10 doesn't have the long supported life

Reply:
Done. PR updated